src/gui/transferlistwidget.cpp

```diff
@@ -198,6 +199,20 @@ namespace
             setDefaultWidget(new MenuCheckBox(text, onToggle, initialState));
         }
     };
+
+    bool torrentContainsPreviewableFiles(const BitTorrent::TorrentHandle *const torrent)
+    {
+        if (!torrent->hasMetadata())
+            return false;
+
+        for (int i = 0; i < torrent->filesCount(); ++i) {
+            if (Utils::Misc::isPreviewable(Utils::Fs::fileExtension(torrent->fileName(i)))) {
```

Comment:
I would remove braces in `if` statement.

Reply:
Done.